manifests/03_operator_rbac.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-storage-operator-role
+  namespace: openshift-cluster-storage-operator
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+subjects:
+  - kind: ServiceAccount
+    name: cluster-storage-operator
+    namespace: openshift-cluster-storage-operator
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Should we fix this and have a fine-grained permissions for this operator? Maybe in a subsequent PR?

Reply:
Added TODO comment